examples/finnegans_wake_demo/finnegans-wake-demo-testnet.py

```diff
@@ -144,7 +144,7 @@
     # The matter of whether encryption makes the passage more or less readable
     # is left to the reader to determine.  Many data sources (Enricos) can
     # encrypt fot the policy's public key.
-    ciphertext, _signature = enrico.encrypt_message(plaintext)
+    ciphertext = enrico.encrypt_message(plaintext)
```

Comment:
update this as well to be `message_kit` (like in getting started docs)? Same for L156.